lib/livebook/file_system/s3.ex

```diff
@@ -14,6 +14,12 @@ defmodule Livebook.FileSystem.S3 do
           hub_id: String.t()
         }
 
+  @type credentials_t :: %{
```

Comment:
```suggestion
  @type credentials :: %{
```

Reply:
I originally called the type `credentials`, but I wasn't clear on whether a module could contain a type and a function with the same name.